en/components/expansion_panel.md

```diff
@@ -63,9 +65,217 @@ The `IgxExpansionPanelComponent` exposes the `.igx-expansion-panel` class which
 ```
 
 You can see the results below: 
-<div class="sample-container loading" style="height: 440px;">
+<div class="sample-container loading" style="height: 200px;">
     <iframe id="expansion-sample-1-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/expansion-sample-1" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
 <div>
     <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="expansion-sample-1-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
-</div>
\ No newline at end of file
+</div>
+
+### Binding to properties and events
+
+We can add some logic to our component to make it show/hide the description depending on the state of the panel. We can do this by binding the description to the controls `collapsed` property:
+
+```typescript
+// in expansion-panel.component.ts
+import { IgxExpansionPanelComponent } from 'igniteui-angular';
+@Component({
+    ...
+})
+export class ExpansionPanelComponent {
+    @ViewChild(IgxExpansionPanelComponent, {read: IgxExpansionPanelComponent})
+    public panel: IgxExpansionPanelComponent;
+    public readMore: string = 'https://en.wikipedia.org/wiki/Hummingbird';
+}
+```
+
+```html
+<!-- in expansion-component.component.html -->
+...
+    <igx-expansion-panel-header>
+        ...
+        <igx-expansion-panel-description *ngIf="panel.collapsed">
+            One of the smallest birds
+        </igx-expansion-panel-description>
+    </igx-expansion-panel-header>
+```
+
+This way, the description will only be display when the component is collapsed.
+
+If we want to add more complex functionality depending on the component's state, we could also bind to the event emitters.
+```typescript
+// in expansion-panel.component.ts
+
+export class ExpansionPanelComponent {
+    ...
+    public handleExpansion(args: {event: Event}): void {
+        ... // Some functionality
```

Comment:
It will be best to give some visual feedback for the user what is the outcome of this eventHandler like "//Check console" or  else.

Reply:
This is addressed in the samples PR